packages/botonic-react/tests/webchat/storage.test.jsx

```diff
@@ -76,4 +76,17 @@ describe('TEST: storageKey', () => {
     expect(localStorage.getItem('botonicState')).not.toBeNull()
     expect(localStorage.getItem('myCustomKey')).toBeNull()
   })
+
+  it('Inits session correctly with user id (corrupted localStorage)', async () => {
```

Comment:
I would use a test.each to test all scenarios (no user, empty user, only id)
and remove unnecessary user fields from the input session to improve readibility